src/Components/Server/src/Circuits/ICircuitInboundEventHandler.cs

```diff
@@ -0,0 +1,18 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.AspNetCore.Components.Server.Circuits;
+
+/// <summary>
+/// A handler to process inbound circuit events.
+/// </summary>
+public interface ICircuitInboundEventHandler
```

Comment:
Not sure how I feel about "EventHandler" being in the name here, given that `EventHandler` is already a different concept. It's supposed to be read "(inbound event) handler" but it might still be confusing.

Reply:
`IHandleCircuitEvent`? Similar to `IHandleAfterRender`?